lib/perl/Genome/VariantReporting/Command/Wrappers/ModelPairFactory.pm

```diff
@@ -68,58 +68,75 @@ sub get_model_pairs {
-            next;
-        }
+            $validation_build = $validation_models[0]->last_succeeded_build;
+            if ( not $validation_build ) {
+                $self->warning_message('No last succeeded build for followup model (%s). Skipping ROI %s.', $validation_models[0]->__display_name__, $roi);
+                next;
+            }
 
-        my $discovery_build = $discovery_models[0]->last_succeeded_build;
-        if ( not $discovery_build ) {
-            $self->warning_message('No last succeeded build for discovery model (%s). Skipping ROI %s.', $discovery_models[0]->__display_name__, $roi);
-            next;
-        }
+            push @model_pairs, Genome::VariantReporting::Command::Wrappers::ModelPair->create(
+                common_translations => $self->get_common_translations(),
+                discovery => $discovery_build,
+                followup => $validation_build,
+                label => "discovery",
+            );
 
-        my $validation_build = $validation_models[0]->last_succeeded_build;
-        if ( not $validation_build ) {
-            $self->warning_message('No last succeeded build for followup model (%s). Skipping ROI %s.', $validation_models[0]->__display_name__, $roi);
-            next;
+            push @model_pairs, Genome::VariantReporting::Command::Wrappers::ModelPair->create(
+                common_translations => $self->get_common_translations(),
+                discovery => $validation_build,
+                followup => $discovery_build,
+                label => "followup",
+            );
         }
 
-        push @model_pairs, Genome::VariantReporting::Command::Wrappers::ModelPair->create(
-            common_translations => $self->get_common_translations(),
-            discovery => $discovery_build,
-            followup => $validation_build,
-            label => "discovery",
-        );
-
-        push @model_pairs, Genome::VariantReporting::Command::Wrappers::ModelPair->create(
-            common_translations => $self->get_common_translations(),
-            discovery => $validation_build,
-            followup => $discovery_build,
-            label => "followup",
-        );
-
         for my $other_input_vcf_pair (keys %{$self->other_input_vcf_pairs}) {
-            push @model_pairs, Genome::VariantReporting::Command::Wrappers::ModelPairWithInput->create(
+            my %params = (
                 common_translations => $self->get_common_translations(),
                 discovery => $discovery_build,
-                followup => $validation_build,
                 plan_file_basename => "cle_docm_report_TYPE.yaml",
                 label => $other_input_vcf_pair,
                 other_snvs_vcf_input => $self->other_input_vcf_pairs->{$other_input_vcf_pair}->[0],
                 other_indels_vcf_input => $self->other_input_vcf_pairs->{$other_input_vcf_pair}->[1],
             );
+            if ($validation_build) {
+                $params{followup} = $validation_build
```

Comment:
I agree. It would be clearer to move "cle_docm_report_TYPE.yaml" here to match "cle_somatic_docm_report_TYPE.yaml" in downstairs else block. I will make the change.